src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Content Collections (Experimental)
+description: Content collections help organize your Markdown and type-check your frontmatter with schemas.
+i18nReady: false
+---
+
+Content collections help organize your Markdown or MDX and type-check your frontmatter with schemas. Collections may be helpful if you:
+
+- **Plan to use Markdown content in multiple areas** of your site (landing pages, footers, navigation, etc).
+- **Have a growing number of documents** to manage and fetch (e.g. a blog with 20+ posts).
```

Comment:
I recommend we remove this unless we have can say which _features_ of Content Collections make more sense as document count grows/is “large”? Is it just that with larger collections, the benefits of frontmatter consistency grows? Or is their some performance gain?

```suggestion
```

Reply:
OK, since we can't find a bullet point that we're all confident on, let's remove. If there's a benefit that's really not otherwise expressed that is a solid explanation of why/when to use, we can always add one back in!